keyboards/lily58/lite_rev3/keyboard.json

```diff
@@ -0,0 +1,104 @@
+{
+    "keyboard_name": "Lily58 Lite Rev3",
+    "manufacturer": "Liliums",
+    "maintainer": "yuchi",
+    "url": "",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "usb": {
+        "vid": "0x1209",
+        "pid": "0x5801",
+        "device_version": "1.0.0"
+    },
+    "split": {
+        "enabled": true
```

Comment:
```suggestion
        "enabled": true,
        "serial": {
            "driver": "vendor"
        } 
```

Reply:
Is this driver definition for LED WS2812?

Reply:
No, it is for serial communication between the two keyboard halves.

Reply:
I've checked the QMK documentation and the JSON schema but I couldn't find any configuration for serial communication
https://docs.qmk.fm/reference_info_json#split-keyboard
https://github.com/qmk/qmk_firmware/blob/master/data/schemas/keyboard.jsonschema#L727-L858

Reply:
Ah, this is is only supported in develop currently (though, develop merge is in a couple of weeks)

Reply:
I've confirmed that it's been added in the master branch. Thanks!